atmo/templates/atmo/jobs/mails/failed_run_alert_body.txt

```diff
@@ -3,7 +3,7 @@ at approximately {{ alert.run.created_at }} UTC.
 
 You may want to check the logs to see what failed in the Spark job.
 
-{% if alert.reason %}The reason for the failure that AWS reported was: {{ alert.reason }}{% endif %}
+{% if alert.reason_message %}The reason for the failure that AWS reported was: {{ alert.reason_message }} {% if alert.reason_code %}({{ alert.reason_code }}){% endif %}{% endif %}
```

Comment:
Is reason_code just a number? If so, it may be good to specify that it's an error code: `(Error code {{ alert.reason_code }})`